File: Source/StringRelatedOperations/to_lower.c
void toLowerCase(char *s, char *result) {
    for(int i = 0; s[i]; i++){
        if(s[i] >= 'A' && s[i] <= 'Z'){
            result[i] = s[i] - 'A' + 'a';
        }
        else {
            result[i] = s[i];
        }
    }
}

File: Source/StringRelatedOperations/to_lower.js
function toLowerCase(s) {
    let result = '';
    for(let i = 0; i < s.length; i++){
        let code = s.charCodeAt(i);
        if(code >= 65 && code <= 90){
            result += String.fromCharCode(code + 32);
        } else {
            result += s[i];
        }
    }
    return result;
}

let s = "Hello, World!";
s = toLowerCase(s);
console.log(s);  // Outputs: hello, world!
